sys/include/net/gnrc/ipv6/nib/conf.h

```diff
@@ -0,0 +1,191 @@
+#define GNRC_IPV6_NIB_CONF_6LN          (1)
+#else
+#define GNRC_IPV6_NIB_CONF_6LN          (0)
+#endif
+#endif
+
+/**
+ * @brief   enable features for IPv6 routers
+ */
+#ifndef GNRC_IPV6_NIB_CONF_ROUTER
+#if GNRC_IPV6_NIB_CONF_6LR
+#define GNRC_IPV6_NIB_CONF_ROUTER       (1)
+#else
+#define GNRC_IPV6_NIB_CONF_ROUTER       (0)
+#endif
+#endif
+
+/**
+ * @brief    (de-)activate router advertising at interface start-up
+ */
+#ifndef GNRC_IPV6_NIB_CONF_ADV_ROUTER
+#if GNRC_IPV6_NIB_CONF_ROUTER && \
+    (!GNRC_IPV6_NIB_CONF_6LR || GNRC_IPV6_NIB_CONF_6LBR)
+#define GNRC_IPV6_NIB_CONF_ADV_ROUTER   (1)
+#else
+#define GNRC_IPV6_NIB_CONF_ADV_ROUTER   (0)
+#endif
+#endif
+
+/**
+ * @brief   (de-)activate NDP address resolution state-machine
+ */
+#ifndef GNRC_IPV6_NIB_CONF_ARSM
+#define GNRC_IPV6_NIB_CONF_ARSM         (1)
+#endif
+
+/**
+ * @brief    queue packets for address resolution
+ */
+#ifndef GNRC_IPV6_NIB_CONF_QUEUE_PKT
+#if GNRC_IPV6_NIB_CONF_6LN
+#define GNRC_IPV6_NIB_CONF_QUEUE_PKT    (0)
+#else
+#define GNRC_IPV6_NIB_CONF_QUEUE_PKT    (1)
+#endif
+#endif
+
+/**
+ * @brief   handle NDP messages according for stateless address
+ *          auto-configuration (if activated on interface)
+ *
+ * @see [RFC 4862](https://tools.ietf.org/html/rfc4862)
+ */
+#ifndef GNRC_IPV6_NIB_CONF_SLAAC
+#define GNRC_IPV6_NIB_CONF_SLAAC        (1)
+#endif
+
+/**
+ * @brief    handle Redirect Messages
+ */
+#ifndef GNRC_IPV6_NIB_CONF_REDIRECT
+#define GNRC_IPV6_NIB_CONF_REDIRECT     (0)
+#endif
+
+/**
+ * @brief   (de-)activate destination cache
+ */
+#ifndef GNRC_IPV6_NIB_CONF_DC
+#if GNRC_IPV6_NIB_CONF_REDIRECT
+#define GNRC_IPV6_NIB_CONF_DC           (1)
+#else
+#define GNRC_IPV6_NIB_CONF_DC           (0)
+#endif
+#endif
+
+/**
+ * @brief   Multihop prefix and 6LoWPAN context distribution
+ *
+ * @see [RFC 6775, section 8.1](https://tools.ietf.org/html/rfc6775#section-8.1)
+ */
+#ifndef GNRC_IPV6_NIB_CONF_MULTIHOP_P6C
+#define GNRC_IPV6_NIB_CONF_MULTIHOP_P6C (0)
+#endif
+
+/**
+ * @brief   Multihop duplicate address detection
+ *
+ * @see [RFC 6775, section 8.2](https://tools.ietf.org/html/rfc6775#section-8.2)
+ */
+#ifndef GNRC_IPV6_NIB_CONF_MULTIHOP_DAD
+#define GNRC_IPV6_NIB_CONF_MULTIHOP_DAD (0)
+#endif
+/** @} */
+
+/**
+ * @brief   Reset time in milliseconds for the reachability time
+ *
+ * @see [RFC 4861, section 6.3.4](https://tools.ietf.org/html/rfc4861#section-6.3.4)
+ */
+#ifndef GNRC_IPV6_NIB_CONF_REACH_TIME_RESET
+#define GNRC_IPV6_NIB_CONF_REACH_TIME_RESET (7200000U)
+#endif
+
+/**
+ * @brief   Maximum link-layer address length (aligned)
+ */
+#if (GNRC_NETIF_HDR_L2ADDR_MAX_LEN % 8)
+#define GNRC_IPV6_NIB_L2ADDR_MAX_LEN        (((GNRC_NETIF_HDR_L2ADDR_MAX_LEN >> 3) + 1) << 3)
+#else
+#define GNRC_IPV6_NIB_L2ADDR_MAX_LEN        (GNRC_NETIF_HDR_L2ADDR_MAX_LEN)
+#endif
+
+/**
+ * @brief   Number of default routers in the default router list.
+ *
+ * **This number has direct influence on the max. number of default routers**
```

Comment:
prefix with `@note` to highlight this information in doxygen. Is there a special purpose for `**`?